types/oidc-provider/oidc-provider-tests.ts

```diff
@@ -68,6 +68,23 @@ new Provider('https://op.example.com', {
     }),
 });
 
+const theJWKS: JWKS = {
```

Comment:
Please add a new test block rather than changing the existing one. This will showcase old code contiues to work.

Reply:
Added.